icinga2-ansible-web-ui/tasks/icinga2_web_ui_RedHat_install.yml

```diff
@@ -1,43 +1,54 @@
 ---
 - name: Install IDO {{ icinga2_web_ui_ido }} on RedHat OS family
-  yum: name=icinga2-ido-{{ icinga2_web_ui_ido }}
-       state=latest
+  yum:
+    name: icinga2-ido-{{ icinga2_web_ui_ido }}
+    state: latest
 
 - include_tasks: icinga2_web_ui_mysql_RedHat_install.yml
   when: icinga2_web_ui_ido == "mysql"
 
 - name: Install Icinga Web on RedHat OS family
-  yum: name={{ item }}
-       state=latest
-  with_items: "{{ icinga2_web_ui_rpm }}"
+  yum:
+    name: "{{ icinga2_web_ui_rpm }}"
+    state: latest
 
 - name: Create a Database for Icinga Web
-  mysql_db: name={{ icinga2_web_db_dbuser }}
-            state=present
+  mysql_db:
+    name: "{{ icinga2_web_db_dbuser }}"
+    state: present
   register: icinga_web_db
 
 - name: Import IDO Schema in Icinga Web Database on RH6 (only once)
-  mysql_db: name={{ icinga2_web_db_dbuser }}
-            state=import
-            target={{ icinga2_web_mysql_schema_rh_6 }}
-  when: ansible_distribution_major_version == "6" and icinga_web_db.changed == true
+  mysql_db:
+    name: "{{ icinga2_web_db_dbuser }}"
+    state: import
+    target: "{{ icinga2_web_mysql_schema_rh_6 }}"
+  when:
+    - ansible_distribution_major_version == "6"
+    - icinga_web_db.changed == true
 
 - name: Import IDO Schema in Icinga Web Database on RH7 (only once)
-  mysql_db: name={{ icinga2_web_db_dbuser }}
-            state=import
-            target={{ icinga2_web_mysql_schema_rh_7 }}
-  when: ansible_distribution_major_version == "7" and icinga_web_db.changed == true
+  mysql_db:
+    name: "{{ icinga2_web_db_dbuser }}"
+    state: import
+    target: "{{ icinga2_web_mysql_schema_rh_7 }}"
+  when:
+    - ansible_distribution_major_version == "7"
+    - icinga_web_db.changed == true
 
 - name: Create Icinga Web Database User and configure Grants
-  mysql_user: name={{ icinga2_web_db_dbuser }}
-              password={{ icinga2_web_db_dbuser }}
-              state=present
-              priv="{{ icinga2_web_db_dbuser }}.*:GRANT,INSERT,SELECT,UPDATE,DELETE,DROP,CREATE VIEW,INDEX,EXECUTE"
+  mysql_user:
+    name: "{{ icinga2_web_db_dbuser }}"
+    password: "{{ icinga2_web_db_dbuser }}"
+    state: present
+    priv: "{{ icinga2_web_db_dbuser }}.*:GRANT,INSERT,SELECT,UPDATE,DELETE,DROP,CREATE VIEW,INDEX,EXECUTE"
```

Comment:
This was changed in one of the latest accepted PR's , please rebase

Reply:
I'm no Git master, so forgive me if I'm wrong. But these snippets are from the web(1) playbook (which I think should be removed since web 1 is already EOL, but that's another issue). 

However, I think you're refering to https://github.com/Icinga/ansible-playbooks/commit/49fd421723d6856708366b037c2fbadb17438644, one of the 3 PR's that was merged today. It only included changes in icinga2_web2, which I believe I have accounted for in this PR. 

Please see https://github.com/Bitblade/ansible-playbooks/commit/78e0338b58ae4776de11d17098817a5f6a8c5327#diff-7799d63164ec0a636267eaa6bc49ea65R3